apps/researcher/src/app/[locale]/communities/[slug]/actions.ts

```diff
@@ -1,22 +1,35 @@
 'use server';
 
-import {joinCommunity, updateDescription} from '@/lib/community';
+import {joinCommunity, updateCommunity} from '@/lib/community';
 import {revalidatePath} from 'next/cache';
 
 interface UpdateDescriptionAndRevalidateProps {
   communityId: string;
-  communitySlug: string;
+  name: string;
+  slug: string;
   description: string;
+  orcid: string;
```

Comment:
We do not know yet whether we're going to use ORCID IDs. Would it be possible to use a more generic name instead, e.g. `attributionId`?